src/environments.jl

```diff
@@ -0,0 +1,65 @@
+project_status() = read(steal_stdout(Pkg.status), String)
+
+function steal_stdout(f)
+  old = stdout
+  rd, wr = redirect_stdout()
+  try
+    f()
+  finally
+    redirect_stdout(old)
+    close(wr)
+  end
+  return rd
+end
+
+@static if VERSION < v"1.1"
+
+update_project() = @msg updateProject(false)
+handle(()->false, "allProjects")
+
+else
+
+update_project() = @msg updateProject(project_info())
+
+# IDEA: Pkg.project will be useful for this, but it's only available as of v1.4
+function project_info()
+  m = match(r"Status \`(?<path>.+)\`$"m, project_status())
+  m === nothing && return false
+  return project_info(m[:path])
+end
```

Comment:
Do you think is it okay to drop support for <v1.4 ? Then we can just use `Pkg.project` for `project_info` implementation and don't need to use the maybe-non-robust regex match.

Reply:
in `project_info` I also use `Pkg.REPLMode.projname`, which is so internal, and I'm not sure from which version those all the Pkg.jl functions are available.
I would like to drop support for <v1.3.